src/components/PlayerInfo/VoteCard/VoteCard.jsx

```diff
@@ -3,7 +3,7 @@ import styles from '../PlayWordAndCard/PlayWordAndCard.module.css';
 
 export const VoteCard = () => {
     return (
-        <div className={styles.playerInteractions}>
+        <div className={styles.playerInteractionsting}>
```

Comment:
Is this a typo?

Reply:
no i needed to detach it from player interactions so i added ting on the end but can change it to something more descriptive

Reply:
The `styles.playerInteractions` was probably adding other styles so by changing the name you might break things that Sisi has done with the css, if you want to add a new class for your own styling then you should make another class with a descriptive name and add that as well, can show you how if you struggle